src/lib/ecs/world.ts

```diff
@@ -191,7 +191,7 @@ export class World {
 	 */
 	public createQuery(
 		arg: RawQuery | AnyComponent | TagComponent,
-		...raw: Array<RawQuery>
+		...raw: Array<RawQuery | AnyComponent | TagComponent>
 	): Query {
```

Comment:
Confused, why the multiplicity in the first place? Can't you just do the following?
```ts
public createQuery(
    ...raw: Array<RawQuery | AnyComponent | TagComponent>
): Query {
```

Reply:
It's to force a single value. Because of the ... this could currently allow no values which would error.